.gitignore

```diff
@@ -69,5 +69,6 @@ tests/BC*
 compile_on_geo.sh
 documentation/tutorials/*/*.html
 pecan.Rproj
+shiny/BenchmarkReport/*
```

Comment:
shiny/BenchmarkReport exists in the repo already, so a commit that adds it to the ignore seems like a bad idea. Could you remove that. Better yet, if you've made changes to BenchmarkReport could you commit those changes too?